core/controller/src/main/scala/whisk/core/controller/Packages.scala

```diff
@@ -60,6 +58,9 @@ trait WhiskPackagesApi extends WhiskCollectionAPI with ReferencedEntities {
   /** JSON response formatter. */
   import RestApiCommons.jsonDefaultResponsePrinter
 
+  /** Reserved package names. */
+  protected[core] val RESERVED_NAMES = Set("default")
```

Comment:
Should we hardcode to “default” or make it configurable, with “default” as ... default :smile:

Reply:
I confused this a bit with the “system” NS. I don’t have a strong pref, nor do I see cases where an operator would wanna change the “default” package. You can Ignore my prev comment.

Reply:
The rationale for reserving `default` is that it's tied to web actions in the "default" package. An explicitly named package with that name would be unreachable.